qdrant-landing/themes/qdrant-2024/layouts/partials/table-contents.html

```diff
@@ -0,0 +1,41 @@
+{{ with (.Site.GetPage "/headless/table-contents") }}
```

Comment:
We need to generate this from content, not from a markdown file. Here is an example: `qdrant-landing/themes/qdrant/layouts/partials/table_of_content.html`.
The `.TableOfContents` method returns an HTML list of content headings:
``` html
<nav id="TableOfContents">
  <ul>
    <li>
      <a href="/aaa">Heading h1</a>
    </li>
    <!-- and so on -->
  </ul>
</nav>
```
[Example here](https://qdrant.tech/documentation/)